src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import core.basesyntax.classes.Color;
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
```

Comment:
Please do not use static methods in your solution.
Static methods are generally much harder to maintain and test so should be used only when static is required